test/test.expressions.js

```diff
@@ -455,6 +455,12 @@ describe('Twig.js Expressions ->', function () {
             output.should.equal('23');
         });
 
+        it('should support slice shorthand (full form) with variables as arguments', function () {
```

Comment:
Please add more test cases to cover both the optional first and second parameters.